File: 3BIN/BINV3140-1 Programmation - question spéciales/BINV3140-A .NET/Séance 1-20200919/3. Solution/BDActeursSolution/BDActeursSolution/ConsoleApplication1/Personne.cs
using System;

namespace be.ipl.film_database.domaine
{
    [Serializable]
    class Personne
    {

        private readonly String _nom;
        private readonly String _prenom;
        private readonly DateTime _dateDeNaissance;

        public virtual String GetNom()
        {
            return _nom;
        }

        public String GetPrenom()
        {
            return _prenom;
        }

        public DateTime GetDateDeNaissance()
        {
            return _dateDeNaissance;
        }

        public Personne(String nom, String prenom, DateTime dateDeNaissance)
        {
            _nom = nom;
            _prenom = prenom;
            _dateDeNaissance = dateDeNaissance;
        }


        public override String ToString()
        {
            return "Personne [nom=" + _nom + ", prenom=" + _prenom
                    + ", dateDeNaissance=" + _dateDeNaissance + "]";
        }

    }
}

File: 3BIN/BINV3140-1 Programmation - question spéciales/BINV3140-A .NET/Séance 1-20200919/2. Exercices/BDActeurs/BDActeurs/src/be/ipl/film_database/domaine/Personne.java
package be.ipl.film_database.domaine;

import java.io.Serializable;
import java.util.Calendar;

public class Personne implements Serializable {
	
	/**
	 * 
	 */
	private static final long serialVersionUID = 1L;
	private final String nom;
	private final String prenom;
	private final Calendar dateDeNaissance;
	
	public String getNom() {
		return nom;
	}
	
	public String getPrenom() {
		return prenom;
	}
	
	public Calendar getDateDeNaissance() {
		return dateDeNaissance;
	}
	
	public Personne(String nom, String prenom, Calendar dateDeNaissance) {
		this.nom = nom;
		this.prenom = prenom;
		this.dateDeNaissance = dateDeNaissance;
	}

	@Override
	public String toString() {
		return "Personne [nom=" + nom + ", prenom=" + prenom
				+ ", dateDeNaissance=" + dateDeNaissance + "]";
	}
	
	
}
